src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -103,14 +104,20 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
     () =>
       spacesMap.map((space) => {
         return (
-          <div key={space.id} onClick={() => setSelectedSpace(space)}>
+          <div key={space.id} data-dropdown-value={space}>
             {space.name}
           </div>
         );
       }) ?? [],
     [spacesMap]
   );
 
+  const selectSpace = (option: Option) => {
+    if (option.value) {
+      setSelectedSpace(option.value as SpaceType);
+    }
+  };
```

Comment:
[Q] Does this mean once selected a space can't be un-selected? How does or should one be able to remove any space from the options?

A second, maybe related, scenario: user picks a space, later deletes that space, what id shown here?

Reply:
The space is required and once selected it makes no sense to unselect it.

We haven't handled this delete case or any other similar ones on the admin side.

Reply:
good enough